src/me/mrCookieSlime/Slimefun/utils/Constants.java

```diff
@@ -0,0 +1,13 @@
+package me.mrCookieSlime.Slimefun.utils;
+
+import me.mrCookieSlime.Slimefun.SlimefunPlugin;
+import org.bukkit.NamespacedKey;
+
+public final class Constants {
+
+    // Note the naming convention for Namespaces is snake_case
+
+    public static final NamespacedKey SF_ITEM = new NamespacedKey(SlimefunPlugin.instance, "sf_item");
```

Comment:
I think instead of creating this memory leaking constants class, just put it into SlimefunPlugin, make it not static and add a static method getNamespacedItemKey() or so.

Reply:
I mean, people shouldn't reload to begin with and then Bukkit should handle it properly... *sigh*

Either way, it's 16 bytes that can leak and for the sake of constants being in one place along with not fulling SlimefunPlugin more. I think it's fine. It's already a 450 line class.